docs/library/machine.I2C.rst

```diff
@@ -112,19 +112,21 @@ control over the bus, otherwise the standard methods (see below) can be used.
 
    Availability: ESP8266.
 
-.. method:: I2C.readinto(buf)
+.. method:: I2C.readinto(buf, nack=True)
```

Comment:
I'll go crawl into a corner now...

I was mis-remembering a picture from the SMBus spec and when I went and looked at it carefully, of course, it matches up with the I2C spec as well. So you are absolutely correct.

The receiver always sends the ACK/NAK (regardless of master/slave).